ddtrace/llmobs/_evaluators/ragas/faithfulness.py

```diff
@@ -196,7 +108,7 @@ def evaluate(self, span_event: dict) -> Tuple[Union[float, str], Optional[dict]]
         If the ragas faithfulness instance does not have `llm` set, we set `llm` using the `llm_factory()`
         method from ragas which defaults to openai's gpt-4o-turbo.
         """
-        self.ragas_faithfulness_instance = _get_faithfulness_instance()
+        self.ragas_faithfulness_instance = self._get_faithfulness_instance()
```

Comment:
Is this still necessary since this already happens in the __init__()?

Reply:
It's an edge case, but same situation as context precision https://github.com/DataDog/dd-trace-py/pull/11716#discussion_r1900178143